app/src/hooks/activeCall.ts

```diff
@@ -3,7 +3,7 @@ import { useSelector } from 'react-redux';
 import { match, P } from 'ts-pattern';
 import { type State } from '../store/app';
 import { type ActiveCall } from '../store/voice/call/activeCall';
-import { setTimeout } from '../util/setTimeout';
+import { setTimeout, requestAnimationFrame } from '../util/setTimeout';
```

Comment:
`requestAnimationFrame` does not seem to be used in this file. Maybe you can remove it?

Reply:
We use it on line `132`. The reason there's no change is because it's also a global, and the one on line `132` used to refer to said global. But now I want to dependency inject because Jest can't seem to replace `requestAnimationFrame` properly.